csharp/Program.cs

```diff
@@ -2,11 +2,48 @@
 
 namespace csharp
 {
+    class Test
```

Comment:
This class name it's ok for a brief test running your code locally, however, if this is going to be merged to a master or dev branch, name needs to reflect (similar to variable and method names) what are the instances of this class going to do